runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonConfig.java

```diff
@@ -28,6 +28,8 @@
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
 
+import java.awt.*;
```

Comment:
Do not use star imports, best would be setup IDEA (if you are using it) with imports like this to prevent future problems with imports: https://github.com/runelite/runelite/wiki/Code-Conventions